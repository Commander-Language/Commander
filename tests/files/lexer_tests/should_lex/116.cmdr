//Should lex function command
fn hello(a, b) echo hi;