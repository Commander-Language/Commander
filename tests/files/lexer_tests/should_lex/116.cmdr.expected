FN 'fn' 2:1
VARIABLE 'hello' 2:4
LPAREN '(' 2:9
VARIABLE 'a' 2:10
COMMA ',' 2:11
VARIABLE 'b' 2:13
RPAREN ')' 2:14
CMDSTRINGVAL 'echo' 2:16
CMDSTRINGVAL 'hi' 2:21
SEMICOLON ';' 2:23
END_OF_FILE 2:24
